kontrol-service/engine/docker.go

```diff
@@ -259,21 +327,60 @@ func newStatefulPluginsAndExternalServicesFromServiceConfig(serviceConfig apityp
 	return serviceStatefulPlugins, externalServices, externalServiceDependencies, nil
 }
 
-func newClusterTopologyServiceFromServiceConfig(serviceConfig apitypes.ServiceConfig, version string) resolved.Service {
+func newClusterTopologyServiceFromServiceConfig(
```

Comment:
I think we can rename this function name to `newClusterTopologyServiceFromConfigs` now that it receives several kind of configs